src/components/visible-while-frozen.js

```diff
@@ -0,0 +1,19 @@
+AFRAME.registerComponent("visible-while-frozen", {
+  init() {
+    this.onStateChange = evt => {
+      if (!evt.detail === "frozen") return;
```

Comment:
I hate how common a pattern this is becoming for us, but it's all for the same reason and should be pretty easy to find and replace once we do something better so...